packages/quantic/cypress/integration/breadcrumb-manager/breadcrumb-manager.cypress.ts

```diff
@@ -0,0 +1,205 @@
+
+          Actions.clickShowMoreNumericFacetBreadcrumb();
+
+          Expect.numerciFacet.numberOfValues(6);
+          Expect.numerciFacet.displayShowMore(false);
+        });
+
+        scope('when clearing a value', () => {
+          Actions.clickFirstValueNumericFacetBreadcrumb();
+          cy.wait(InterceptAliases.Search);
+
+          Expect.numerciFacet.numberOfValues(5);
+        });
+      });
+
+      scope('when selecting values in time frame facet', () => {
+        Expect.dateFacet.displayBreadcrumb(false);
+
+        Actions.timeFrameFacet.selectValue('Past 6 months');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.dateFacet.displayBreadcrumb(true);
+        Expect.dateFacet.displayLabel(true);
+        Expect.dateFacet.labelContains('Date');
+        Expect.dateFacet.numberOfValues(1);
+        Expect.dateFacet.breadcrumbValueLabelAtIndexContains('Past 6 months');
+
+        Actions.timeFrameFacet.selectValue('Past year');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.dateFacet.displayBreadcrumb(true);
+        Expect.dateFacet.displayLabel(true);
+        Expect.dateFacet.numberOfValues(1);
+        Expect.dateFacet.breadcrumbValueLabelAtIndexContains('Past year');
+      });
+
+      scope('when clearing all filters', () => {
+        Actions.clickClearFilters();
+
+        Expect.numerciFacet.displayBreadcrumb(false);
+        Expect.dateFacet.displayBreadcrumb(false);
+      });
+
+      scope('when selecting values in category facet', () => {
+        Expect.categoryFacet.displayBreadcrumb(false);
+
+        Actions.categoryFacet.selectChildValue('North America');
+
+        Expect.categoryFacet.displayBreadcrumb(true);
+        Expect.categoryFacet.displayLabel(true);
+        Expect.categoryFacet.labelContains('Country');
+        Expect.categoryFacet.displayValues(true);
+        Expect.categoryFacet.numberOfValues(1);
+        Expect.categoryFacet.breadcrumbValueLabelAtIndexContains(
+          'North America'
+        );
+
+        Actions.categoryFacet.selectChildValue('Canada');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.categoryFacet.numberOfValues(1);
+        Expect.categoryFacet.breadcrumbValueLabelAtIndexContains(
+          'North America / C'
+        );
+
+        Actions.categoryFacet.selectChildValue('British Columbia');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.categoryFacet.numberOfValues(1);
+        Expect.categoryFacet.breadcrumbValueLabelAtIndexContains(
+          'North America / C'
+        );
+      });
+
+      scope('when selecting values in facet', () => {
+        Actions.clickClearFilters();
+
+        Actions.facet.selectFirstLinkValue();
+
+        Expect.facet.displayBreadcrumb(true);
+        Expect.facet.labelContains('File Type');
+        Expect.facet.displayValues(true);
+        Expect.facet.numberOfValues(1);
+        Expect.facet.breadcrumbValueLabelAtIndexContains('txt');
```

Comment:
We can't assume that the first facet value will be `txt`. It might well change over time for many reasons.

It would seem more reliable to me to retrieve the selected value after calling `Actions.facet.selectFirstLinkValue()`, and then reuse the same value to validate the breadcrumb value label. The downside is that we'd need to call the selector with `.then()` to be able to validate the expectation, and it would make the code a little more complicated here.